akka-projection-kafka/src/it/scala/akka/projection/kafka/integration/KafkaToSlickIntegrationSpec.scala

```diff
@@ -209,8 +207,11 @@ class KafkaToSlickIntegrationSpec extends KafkaSpecBase(ConfigFactory.load().wit
 
     // relates to issue https://github.com/akka/akka-projection/issues/305
     // we must ensure that we can read a kafka offset from the storage
+    //
+    // requires further investigation. a change during build instability broke this test.
+    // https://github.com/akka/akka-projection/issues/341
```

Comment:
Something happened to this test, but not sure what.  I created a follow up issue https://github.com/akka/akka-projection/issues/341